docs/reference/data/understanding-database.md

```diff
@@ -0,0 +1,54 @@
+# Understanding Database
+
```

Comment:
The page would not be accessible via the sidebar as of now. Can you please make the appropriate changes in the config file?